conf/.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+
+# Increase max line length to be the same as python black
+max-line-length = 88
+extend-ignore = E203
```

Comment:
This one is not great, IMO - there should never be whitespace before a `:` and if you're writing slices that look like `[1+1:2+2]`, then you've got bigger problems than formatting.